From c3e39ef14e99f903b95fa051936f40ebd6c35484 Mon Sep 17 00:00:00 2001
From: Auger Eric <eric.auger@redhat.com>
Date: Mon, 3 Jun 2019 19:17:39 +0100
Subject: [PATCH 2/9] aarch64: Add virt-rhel8.1.0 machine type for ARM
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Auger Eric <eric.auger@redhat.com>
Message-id: <20190603191740.32665-2-eric.auger@redhat.com>
Patchwork-id: 88461
O-Subject: [RHEL-AV-8.1.0 qemu-kvm PATCH 1/2] aarch64: Add virt-rhel8.1.0 machine type for ARM
Bugzilla: 1713735
RH-Acked-by: Andrew Jones <drjones@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>

This patch adds a new machine type, virt-rhel8.1.0, for QEMU rhel-av.
This machine type is based off 4.0 with the exception that it removes
support for dynamic sysbus devices: VFIO_CALXEDA_XGMAC, VFIO_AMD_XGBE, and
RAMFB_DEVICE because downstream RHEL doesn't support these devices. IOMMU
instantiation still is disabled but will be enabled in subsequent patches.

Signed-off-by: Eric Auger <eric.auger@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 hw/arm/virt.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hw/arm/virt.c b/hw/arm/virt.c
index 5602d9f..9316a8d 100644
--- a/hw/arm/virt.c
+++ b/hw/arm/virt.c
@@ -2158,7 +2158,7 @@ static void rhel_machine_init(void)
 }
 type_init(rhel_machine_init);
 
-static void rhel800_virt_instance_init(Object *obj)
+static void rhel810_virt_instance_init(Object *obj)
 {
     VirtMachineState *vms = VIRT_MACHINE(obj);
     VirtMachineClass *vmc = VIRT_MACHINE_GET_CLASS(vms);
@@ -2204,8 +2204,8 @@ static void rhel800_virt_instance_init(Object *obj)
     vms->irqmap=a15irqmap;
 }
 
-static void rhel800_virt_options(MachineClass *mc)
+static void rhel810_virt_options(MachineClass *mc)
 {
     compat_props_add(mc->compat_props, arm_rhel_compat, arm_rhel_compat_len);
 }
-DEFINE_RHEL_MACHINE_AS_LATEST(8, 0, 0)
+DEFINE_RHEL_MACHINE_AS_LATEST(8, 1, 0)
-- 
1.8.3.1

